.github/workflows/release.yaml

```diff
@@ -33,9 +33,15 @@ jobs:
           go-version-file: go.mod
 
       - name: Run Go tests
+        env:
+          # Test with the same environment as the build.
+          CGO_ENABLED: 0
```

Comment:
I’d prefer to run tests both with and without `CGO_ENABLED`. Can you add a new step below this named `Run Go tests without CGO`?

Reply:
Never mind. The test action runs this already.

This looks fine.